bin/file_extension.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python3
+
+import sys
+
+from utils import json_dump, package_validation_dict
+
+
+def file_extension_test(file_path: str) -> str:
+    """
+    1. Reads file extension from provided path
+    2. Checks for valid file extension (.txt is used for now, future version will use ome.tiff and ome.tif)
+
+    Args:
+        file_path (str): Path to the downloaded file
+
+    Returns:
+        str: Result of File extension test
+    """
+    ext = file_path.split(".", 1)[-1]
+    # Do we want the extension check to be hard-coded like this or check against the file type data provided in the CSV?
+    # We are not using that column of the CSV at the moment.
+    file_ext_test = (ext == "ome.tiff" or ext == "ome.tif") 
+    file_ext_status = "pass" if file_ext_test else "fail"
+    return file_ext_status
+
+
+if __name__ == '__main__':
+    syn_id = sys.argv[1]
+    entity_type = sys.argv[2]
+    version_number = sys.argv[3]
+    md5_input = sys.argv[4]
+    file_path = sys.argv[5]
+    file_ext_status = file_extension_test(file_path)
+    file_extension_check_data = package_validation_dict(
+    syn_id=syn_id, 
+    entity_type=entity_type, 
+    version_number=version_number, 
+    md5_input=md5_input, 
+    validation_status=file_ext_status,
+    test_name="file_extension_test"
+    )
```

Comment:
**Nit (_i.e._ [optional](https://google.github.io/eng-practices/review/reviewer/standard.html#:~:text=Reviewers%20should%20always%20feel%20free,they%20could%20choose%20to%20ignore.)):** The lines in parentheses should be indented. I recommend enabling auto-formatting in VS Code using the `black` package to have this done automatically every time you save the file ([docs](https://dev.to/adamlombard/how-to-use-the-black-python-code-formatter-in-vscode-3lo0)).